test/parallel/test-util-inspect.js

```diff
@@ -832,16 +838,22 @@ checkAlignment(new Map(big_array.map(function(y) { return [y, null]; })));
 // Do not backport to v5/v4 unless all of
 // https://github.com/nodejs/node/pull/6334 is backported.
 {
-  const x = Array(101);
+  const x = new Array(101).fill();
   assert(/1 more item/.test(util.inspect(x)));
 }
 
 {
-  const x = Array(101);
+  const x = new Array(101).fill();
   assert(!/1 more item/.test(util.inspect(x, {maxArrayLength: 101})));
 }
 
 {
+  const x = new Array(101).fill();
+  assert(/^\[ ... 101 more items ]$/.test(
+      util.inspect(x, {maxArrayLength: 0})));
```

Comment:
nit: I know the regex is there already, but it doesn't seem to be needed, since `assert.strictEqual(util.inspect(x, {maxArrayLength: 0}), '[ ... 101 more items ]')` should accomplish the same.